### PR TITLE
PurchaseTester: defined APIKey in a common file

### PR DESCRIPTION
**Pull request by NachoSoto**
This simplifies the error when initially compiling the project and points to a single file to fix it.

Screen Shot 2021-11-15 at 15 18 12

### CODE REVIEW

**Comment by NachoSoto** [comment]
I see that `Fastfile` replaces these for other integration test projects, but looks like not this one. Let me know if I missed something though.

File: IntegrationTests/PurchaseTester/PurchaseTester.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -26,6 +26,8 @@
 		3493336B21E69360007AAAB9 /* LaunchScreen.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = 3493336921E69360007AAAB9 /* LaunchScreen.storyboard */; };
 		3493337521E694D6007AAAB9 /* CatsViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 3493337421E694D6007AAAB9 /* CatsViewController.swift */; };
 		34D05817233C1A3700343134 /* SwiftPaywall.swift in Sources */ = {isa = PBXBuildFile; fileRef = 34D05816233C1A3700343134 /* SwiftPaywall.swift */; };
+		5791CEC62743219A00E50C4B /* Constants.swift in Sources */ = {isa = PBXBuildFile; fileRef = 5791CEC52743219A00E50C4B /* Constants.swift */; };
```
Comment (taquitos) [inline IntegrationTests/PurchaseTester/PurchaseTester.xcodeproj/project.pbxproj:29]: Double add in the project file

**Review by taquitos — APPROVED** [review]
🔑🦕 👍

File: IntegrationTests/PurchaseTester/PurchaseTester.xcodeproj/project.pbxproj (hunk 1 above)
Comment (NachoSoto) [reply IntegrationTests/PurchaseTester/PurchaseTester.xcodeproj/project.pbxproj:29]: It's in the main target and the watch extension :)

Comment (NachoSoto) [reply IntegrationTests/PurchaseTester/PurchaseTester.xcodeproj/project.pbxproj:29]: I think? Let me double check.

Comment (taquitos) [reply IntegrationTests/PurchaseTester/PurchaseTester.xcodeproj/project.pbxproj:29]: Ohhh that makes sense.

Comment (NachoSoto) [reply IntegrationTests/PurchaseTester/PurchaseTester.xcodeproj/project.pbxproj:29]: Yup confirmed.

**Pull request merged**